Engines/Wine/QuickScript/Zip Script/script.js

```diff
@@ -57,7 +57,8 @@ ZipScript.prototype.go = function () {
         .wizard(setupWizard)
         .archive(archive)
         .to(wine.prefixDirectory() + "/drive_c/" + this._name)
-        .extract();
+        .extract()
```

Comment:
There is a `;` missing after `extract()`. Please also add an empty line between this and `wine.runInside...`. This makes the code better readable.